lib/components/map/default-map.tsx

```diff
@@ -250,7 +250,8 @@ class DefaultMap extends Component {
 
     const bikeStations = [
       ...bikeRentalStations.filter(
-        (station) => station.isFloatingVehicle === false
+        (station) =>
+          !station.isFloatingVehicle || station.isFloatingVehicle === false
```

Comment:
In some cases `isFloatingVehicle` is undefined, in some cases it's explicitly false. It should be caught in the first condition, but this fix resolved issues I was having that wouldn't be sorted otherwise. I agree it's strange

Reply:
Just something fun for future developers to find and wonder about.